be/src/vec/data_types/data_type_factory.hpp

```diff
@@ -95,7 +97,10 @@ class DataTypeFactory {
         });
```

Comment:
register Map to _invert_data_type_map like Array

Reply:
May be we will add this part in next map pr.